dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -241,7 +243,7 @@ public String buildInsertCommand(String tableName,
     }
 
     public DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors) {
```

Comment:
Now I'm thinking...couldn't this method just return a `StatementExecution` directly instead of a new `DbStoredProcedureCall` to `DbObject.buildPreparedStatement`?

I'll push an experimental commit to review.